components/LabeledLinkButton.jsx

```diff
@@ -0,0 +1,22 @@
+import StyledShowMore from '../components-styled/StyledShowMore';
+import StyledSvgWrapper from '../components-styled/StyledSvgWrapper';
+import Link from 'next/link';
+
+export default function LabeledLinkButton({
+  label,
+  target,
+  children,
+  onClick,
+  className,
+}) {
+  return (
+    <Link passHref href={`/${target}`}>
+      <StyledShowMore onClick={onClick} className={`${className}`}>
```

Comment:
When you use a Link component you need to have an anchor element as its direct child. So you could use an anchor instead of div for the ShowMore

Reply:
done